module/cmd/gravity/cmd/testnet.go

```diff
@@ -171,7 +171,7 @@ func InitTestnet(
 		gentxsDir := filepath.Join(outputDir, "gentxs")
 
 		nodeConfig.SetRoot(nodeDir)
-		nodeConfig.RPC.ListenAddress = "tcp://0.0.0.0:26657"
+		nodeConfig.RPC.ListenAddress = "tcp:// 0.0.0.0:26657"
```

Comment:
won't this cause an error? Oh but I see it's required by the lint, that's crazy. We should disable that one probably.

Reply:
This was definitely a regex failure, it cares about comments not strings